### ISSUE TITLE
change link from @warsow to qfusion

### ISSUE THREAD

**Issue by Vectory** [leereilly/games#295]
The link for warsow doesn't go to the official game code, or any repo at all, just a user hosting some game extras. The contributers are associated devs, however.

The engine is still developed under https://github.com/Qfusion/qfusion/ - I think the dev Vic parted ways and changed the name repo's name. With a lot of community drama that happened, I don't know what's the actual situation. The game's web page is still up and the engine is still in development, going by the commits.

**Comment by mdtrooper** [comment]
Yes, it sounds true as the oficial webpage says in https://www.warsow.net/wiki/Second%20Dev-Wiki
